app/src/pages/inside/common/infoPanel/viewTabs/viewTabs.jsx

```diff
@@ -134,21 +138,31 @@ export class ViewTabs extends Component {
   };
 
   render() {
-    const { viewMode } = this.props;
+    const {
+      viewMode,
+      intl: { formatMessage },
+    } = this.props;
     const pages = this.getPages();
 
     return (
       <div className={cx('view-tabs')}>
-        {pages.map((page) => (
-          <Link
-            key={page.id}
-            to={page.link}
-            className={cx('view-tab-link', { active: viewMode === page.id })}
-          >
-            {page.icon && <i className={cx('icon')}>{Parser(page.icon)}</i>}
-            {page.title}
-          </Link>
-        ))}
+        {pages.map((page) => {
+          const isUniqueErrorsViewTab = page.id === UNIQUE_ERRORS_VIEW;
+          return (
+            <Link
+              key={page.id}
+              to={page.link}
+              title={isUniqueErrorsViewTab && formatMessage(messages.uniqueErrorsViewTooltip)}
```

Comment:
This needs to be done at the configuration level - it would be a bit odd to add such conditions to other pages if needed in the future.
The best solution here is to provide additional properties to `pages` array (`disabled` and `hint`) for `UNIQUE_ERRORS_VIEW` item.